packages/api/package.json

```diff
@@ -6,12 +6,11 @@
   "type": "module",
   "main": "dist/worker.js",
   "scripts": {
-    "lt": "npm run lt:cluster & npm run lt:ipfs-proxy & npm run lt:postgrest",
-    "lt:cluster": "npx localtunnel --port 9094 --subdomain \"$(whoami)-cluster-api-nft-storage\"",
-    "lt:ipfs-proxy": "npx localtunnel --port 9095 --subdomain \"$(whoami)-ipfs-proxy-api-nft-storage\"",
-    "lt:postgrest": "npx localtunnel --port 3000 --subdomain \"$(whoami)-postgrest-nft-storage\"",
     "deploy": "wrangler publish --env production",
-    "dev": "wrangler dev",
+    "predev": "./scripts/cli.js db --start && ./scripts/cli.js db-sql --cargo --testing --reset",
+    "dev": "miniflare --watch --debug --env ../../.env",
+    "clean": "./scripts/cli.js db --clean && rm -rf db/docker/compose",
```

Comment:
let's add a "postdev" script for cleanup?

Reply:
doesn't work because dev stay running we kill it manually, so i thought `clean` is more explicit

Reply:
but, once we force exit miniflare (CTRL+C), we need  to run clean, right?

Reply:
or let it run in the background depends on what you are doing 

maybe we can also add a stop, the cli already has it `./scripts/cli.js --stop`